src/main/java/uk/gov/hmcts/reform/sendletter/queue/producer/AzureQueueProducerClient.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.reform.sendletter.queue.producer;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.azure.servicebus.IQueueClient;
+import com.microsoft.azure.servicebus.Message;
+import com.microsoft.azure.servicebus.QueueClient;
+import com.microsoft.azure.servicebus.ReceiveMode;
+import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.sendletter.model.Letter;
+
+import java.time.Duration;
+import java.util.Random;
+
+@Component
+public class AzureQueueProducerClient implements QueueProducerClient {
+
+    private static final Logger log = LoggerFactory.getLogger(AzureQueueProducerClient.class);
+
+    private final ObjectMapper objectMapper;
+
+    private final String connectionString;
+
+    private final String queueName;
+
+    private final String messageTimeToLive;
+
+    public AzureQueueProducerClient(ObjectMapper objectMapper,
+                                    @Value("${servicebus.connectionString}") String connectionString,
+                                    @Value("${servicebus.queue.name}") String queueName,
+                                    @Value("${servicebus.queue.messageTTLInDays}") String messageTimeToLive) {
+        this.objectMapper = objectMapper;
+        this.connectionString = connectionString;
+        this.queueName = queueName;
+        this.messageTimeToLive = messageTimeToLive;
+    }
+
+    @Override
+    public String send(Letter letter) throws ServiceBusException, InterruptedException, JsonProcessingException {
```

Comment:
add tests

Reply:
sure will add tests.Sorry missed this one